src/zope/app/apidoc/codemodule/browser/README.rst

```diff
@@ -44,7 +44,7 @@ Return info objects for all classes in this module.
   >>> pprint(details.getClasses())
   [{'doc': 'Represent the code browser documentation root',
     'name': 'CodeModule',
-    'url': 'http://127.0.0.1/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
+    'url': 'http://127.0.0.1/++etc++site/++apidoc++/Code/zope/app/apidoc/codemodule/codemodule/CodeModule'}]
```

Comment:
When you say "reparents", do you mean "wraps in a LocationProxy", or is there more of this "heyyyy let's modify the attributes of random shared global objects because YOLO" we've seen in zope.app.onlinehelp?